tools/eos-content-merge

```diff
@@ -136,6 +137,22 @@ class App(object):
             self.verbose_print('Updated desktop file follows:')
             outdesktop.write(sys.stdout, space_around_delimiters=False)
 
+    def merge_appdata_file(self, outpath, inplace=False):
+        shell_content = ShellContent()
+        xml = shell_content.update_appdata_from_file(outpath, self.appid)
+        if not xml:
+            self.verbose_print('Failed to get updated XML from ', outpath)
+            self.verbose_print('Not saving file!')
```

Comment:
If the XML could not be read, shouldn't this error or at least return? The code below expects that the XML was read.